components/style/context.rs

```diff
@@ -364,7 +364,7 @@ impl<E: TElement> SelectorFlagsMap<E> {
     pub fn new() -> Self {
         SelectorFlagsMap {
             map: FnvHashMap::default(),
-            cache: LRUCache::new(4),
+            cache: LRUCache::<[_;4]>::new(),
```

Comment:
I _think_ you don't need the explicit type parameter, and that doing `LRUCache::new()` should be enough. But if you really need it, please put a space after the `;`.